pkg/lib/bundle/utils.go

```diff
@@ -0,0 +1,110 @@
+package bundle
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"syscall"
+)
+
+func copyDirectory(scrDir, dest string) error {
```

Comment:
should we just import https://github.com/gotestyourself/gotest.tools/blob/master/fs/ops.go ?

Reply:
I didn't really want to add a dependency for copying files around and couldn't find a safe/stable repo to rely on. If you think this one works I'm happy to add it

Reply:
Uhh, can we just use code from https://stackoverflow.com/questions/51779243/copy-a-folder-in-go ?